index.js

```diff
@@ -1,13 +1,14 @@
 const Cryptr = require('cryptr');
-const { emit, on } = require('./symbol');
+const { emit, on, off, send } = require('./symbol');
 const reservedEvents = require('./reserved-events');
 
 module.exports = (secret) => (socket, next) => {
+  const handlers = new Map();
```

Comment:
Wondering if this will work with [WeakMap](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap)? Just curious… might help prevent possible [memory leaks](https://stackoverflow.com/questions/49262651/how-map-in-javascript-cause-memory-leak-while-weakmap-is-not).